src/lib/tvgCanvasImpl.h

```diff
@@ -59,11 +59,9 @@ struct Canvas::Impl
         if (!renderer || !renderer->clear()) return Result::InsufficientCondition;
 
         //free paints
-        if (free) {
-            for (auto paint = paints.data; paint < (paints.data + paints.count); ++paint) {
-                (*paint)->pImpl->dispose(*renderer);
-                delete(*paint);
-            }
+        for (auto paint = paints.data; paint < (paints.data + paints.count); ++paint) {
+            (*paint)->pImpl->dispose(*renderer);
```

Comment:
I think this part could cause crash in sequence: tvg_canvas_clear(canvas, false); tvg_paint_del(paint); We are not able to check easily if canvas was cleared and then ignore tvg_paint_del. Also we are not able to ignore tvg_paint_del() completely, because paint duplication cause memory leaks.

Reply:
Can you explain what do you prefer the usage here? 
do you want this?
tvg_canvas_clear(canvas, false);
tvg_paint_del(paint);

Reply:
see https://github.com/Samsung/thorvg/pull/382